x-pack/plugins/triggers_actions_ui/public/application/lib/validate_params_for_warnings.ts

```diff
@@ -13,7 +13,8 @@ import Mustache from 'mustache';
 const publicUrlWarning = i18n.translate(
   'xpack.triggersActionsUI.sections.actionTypeForm.warning.publicUrl',
   {
-    defaultMessage: 'server.publicBaseUrl is not set. Actions will use relative URLs.',
+    defaultMessage:
+      'server.publicBaseUrl is not set. Generated URLs will be either relative or empty.',
```

Comment:
Do you need to change `x-pack/plugins/triggers_actions_ui/public/application/lib/validate_params_for_warnings.test.ts` tests as well according that change?

Reply:
Yes, thank you!